enterprise/api/v1/views.py

```diff
@@ -138,10 +168,26 @@ def get_serializer_class(self):
         return serializers.EnterpriseCourseEnrollmentWriteSerializer
 
 
+class EnterpriseCustomerCatalogViewSet(EnterpriseReadOnlyModelViewSet):
+    """
+    API views for the ``enterprise_catalog`` API endpoint.
```

Comment:
@UmanShahzad We still have an underscore here. Should probably turn it into a dash after fbdf6222b8925de0ed2b4d2d302d014fb03fbdd4.